src/containers/SidechainHome/sidechainhome.scss

```diff
@@ -0,0 +1,126 @@
+@import '../shared/css/variables.scss';
+
+.app {
+  position: relative;
+
+  .sidechain-main-page {
+    position: relative;
+    min-height: 600px;
+    color: $white;
+
+    .logo-content {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+      padding: auto;
+      margin: 97px auto 80px;
+      text-align: center;
+
+      .sidechain-logo {
+        width: auto;
+        height: auto;
+
+        @include for-size(tablet-landscape-up) {
+          width: 400px;
+        }
+      }
+
+      .page-header {
+        margin: 0px 24px;
```

Comment:
nit - by convention in css if the value is 0 you don't specify a unit, i believe